projects/urls.py

```diff
@@ -5,9 +5,10 @@
 from django.conf.urls.static import static
 
 urlpatterns = [
-    path("admin/doc/", include("django.contrib.admindocs.urls")),
-    path("admin/", admin.site.urls),
+    # path("admin/doc/", include("django.contrib.admindocs.urls")), # Check error
+    # path("admin/", admin.site.urls), # Check error
     path("media_server/", include("openedx_learning.contrib.media_server.urls")),
     path("rest_api/", include("openedx_learning.rest_api.urls")),
-    path('__debug__/', include('debug_toolbar.urls')),
+    path("rest_api2/", include("openedx_tagging.core.tagging.urls")),
```

Comment:
Let's put this under a different URL, and add a namespace:
```suggestion
    path("tagging/rest_api/", include("openedx_tagging.core.tagging.urls", namespace="oel_tagging")),
```

This change (plus the [added `app_name` change suggested above](https://github.com/open-craft/openedx-learning/pull/4/files#diff-c1ed6a4e1641edb5b1f7ad4f61be866a45c8784463eff6b7d7f54adc8775d891R8)) mean you need to prefix the URL names requested by `reverse()` with this namespace:

```
url = reverse("oel_tagging:taxonomy-list")
```